Core/DISET/private/MessageBroker.py

```diff
@@ -8,7 +8,7 @@
 
 from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.Core.DISET.private.TransportPool import getGlobalTransportPool
-from DIRAC.Core.Utilities.ThreadPool import getGlobalThreadPool
+from DIRAC.Core.Utilities.ThreadPool import getGlobalThreadPoolExecutor
```

Comment:
Is this function defined ? in the ThreadPool.py ?

Reply:
No. But this is already fixed. I am working on it, it will change...